gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -41,6 +41,7 @@
 import com.google.common.collect.ImmutableList;
 import io.grpc.ManagedChannel;
 import io.grpc.ManagedChannelBuilder;
+import io.grpc.netty.shaded.io.grpc.netty.NettyChannelBuilder;
```

Comment:
We need to suport both grpc-netty-shaded and grpc-netty. To do that, we need to check for the existence of both instances of `NettyChannelBuilder` (using reflection).